cache.go

```diff
@@ -45,6 +45,14 @@ func (cache Cache) Store(configpath string) {
 	}
 }
 
+func CacheLastModified(configpath string) (time.Time, error) {
+	stat, err := os.Stat(fmt.Sprintf("%s/cache", configpath))
```

Comment:
Stat:ing feels a bit iffy to me. But adding some last updated timestamp to the cache file would imply changing its format, too... hmm

Reply:
As stated above I _think_ this is an acceptable approach IHMO.